eq-author-api/middleware/identification/index.test.js

```diff
@@ -1,5 +1,5 @@
 const identificationMiddleware = require("./");
-const jwt = require("jsonwebtoken");
+const mockJwt = require("jsonwebtoken");
```

Comment:
Where do we mock this?

Reply:
Not mocked but component within jest.mock can only reference modules with names starting with mock. I've changed it so jwt is imported within jest.mock too so that it's clearer whats happening.